docs/first-steps/diving-deeper/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Exploring Snowplow beyond the first steps"
+sidebar_position: 7
+sidebar_label: "Diving deeper"
+description: "What to look at once you are familiar with Snowplow basics"
+---
+
+Here are a few suggestions for your next steps:
+* Browse the [recipes](/docs/recipes/index.md) — short guides for common use cases
+* For more end-to-end guides and templates, check out our [data product accelerators](https://snowplow.io/data-product-accelerators/)
+* Learn more about [defining the shape of your data](/docs/understanding-tracking-design/index.md): tracking design, managing and versioning your schemas
+* Take a look at the [enrichment process](/docs/enriching-your-data/index.md) and the available enrichments, which can add extra information to your data
+* Explore [data models](/docs/modeling-your-data/index.md), which build upon your event data and provide useful aggregations
```

Comment:
Maybe we could move this to the top or otherwise highlight it? I even wonder if (maybe for a v2) deploying the webmodel deserves its own "First steps" page